build/root/usr/local/bin/ci_entrypoint_nto.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+
+set -o pipefail
+set -o errexit
+set -o nounset
+
+nto_e2e() {
+    toolbox/cluster/capture_environment.sh
+    toolbox/nto/run_e2e_test.sh "$@"
+}
+
+if [ -z "${1:-}" ]; then
+    echo "FATAL: $0 expects at least 1 argument ..."
+    exit 1
+fi
+
+action="${1:-}"
+shift
+
+set -x
+
+case ${action:-} in
+    "e2e")
+        nto_e2e "$@"
+        exit 0
+        ;;
+    -*)
+        echo "FATAL: Unknown option: ${action}"
+        exit 1
+        ;;
+    *)
+        echo "FATAL: Unknown action \"${action}\""
+        exit 1
+        ;;
```

Comment:
do you know if there is a difference between these two cases?
`*)` is the default, but `-*)` I'm not sure

Reply:
Not sure I understand the question.  I added the `\"${action}\"` to assist better debugging.  "FATAL: Nothing to do ..." doesn't help much.

`-*)` will catch the actions starting with `-` (i.e. options),
`*)` will catch the rest (empty string, ...).

and at least you'll have a better idea what happened.

Reply:
oh, I though `-*)` was a special bash syntax :# :# :#

now I understand better, I thought my `nothing to do` would only be executed when no parameter was passed, that's why I didn't print the argument value in the message...

```
    -*)
            echo "FATAL: Unknown option: ${target}"     
            exit 1
```
this block is not necessary then, we never expect "options" in the entrypoints, at least in `run`

Reply:
OK, will remove the option's block then. :)